README.md

```diff
@@ -1,20 +1,41 @@
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-technocrats/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-technocrats?branch=develop)
```

Comment:
This is not related to the story referenced in this PR

Reply:
We've worked on your feedback.